### PR TITLE
render policies

### PR DESCRIPTION
**Pull request by JimBugwadia**
Signed-off-by: Jim Bugwadia <jim@nirmata.com>

## Checklist

- [x] I have read the [contributing guidelines](https://github.com/kyverno/website/blob/main/CONTRIBUTING.md).
- [x] I have inspected the website preview for accuracy.
- [x] I have signed off my issue.

### CODE REVIEW

**Comment by chipzoller** [comment]
Needs cherry picking to 1-6-0 also

**Comment by JimBugwadia** [comment]
> Needs cherry picking to 1-6-0 also

Will do, once merged.

**Review by patelrit — APPROVED** [review]

**Review by chipzoller — CHANGES_REQUESTED** [review]
Need to fix the verifyImage policies rendering, probably by rebuilding the `render` program with latest changes.

File: content/en/policies/other/exclude_namespaces_dynamically.md [hunk 1]
```diff
@@ -45,7 +45,7 @@ spec:
     preconditions:
       all:
       - key: "{{request.object.metadata.namespace}}"
-        operator: NotIn
+        operator: AnyNotIn
```
Comment (chipzoller) [inline content/en/policies/other/exclude_namespaces_dynamically.md:48]: If you change this operator to `AnyNotIn` you change minimum version compatibility to 1.6.0, so we must add the appropriate annotation with value. There are many that do need updating, and I consciously avoided this until the next minor version to give users a chance to catch up.

**Review by chipzoller — CHANGES_REQUESTED** [review]
See comment about changing operators to those in 1.6.0.

File: content/en/policies/other/exclude_namespaces_dynamically.md (hunk 1 above)
Comment (JimBugwadia) [reply content/en/policies/other/exclude_namespaces_dynamically.md:48]: Added comment in: https://github.com/kyverno/policies/pull/284

**Comment by JimBugwadia** [comment]
@chipzoller - anything else necessary to merge and cherry-pick?

**Review by chipzoller — APPROVED** [review]
Looks good now.

**Pull request merged**